contracts/Comet.sol

```diff
@@ -477,7 +477,15 @@ contract Comet is CometMainInterface {
     }
 
     /**
-     * @notice Gets the total amount of protocol reserves, denominated in the number of base tokens
+     * @notice Gets the total balance of protocol collateral reserves for an asset
```

Comment:
Worth documenting that we're intending for this to revert if `balanceOf(this) < totalSupplyAsset`?

Reply:
Yep